packages/abstractions/src/apiClientProxifier.ts

```diff
@@ -194,7 +194,11 @@ export const apiClientProxifier = <T extends object>(requestAdapter: RequestAdap
 						if (navigationCandidate.pathParametersMappings && navigationCandidate.pathParametersMappings.length > 0) {
 							for (let i = 0; i < argArray.length; i++) {
 								const element = argArray[i];
-								downWardPathParameters[navigationCandidate.pathParametersMappings[i]] = element;
+								if (element) {
```

Comment:
```suggestion
								if (element === undefined || element === null || element === '') {
```
In case the parameter is a number and somehow there's something special with id zero or something